bubble-wrap-style.yaml

```diff
@@ -3366,7 +2563,19 @@ layers:
                     outline:
                         order: 354
                         color: global.minor_casing1
-                        width: [[15, 0px], [16, 0.5px], [17, 1.0px], [18, 2.0px]]
+                        width: [[14, 0px], [15, 0.2px], [16, 0.5px], [17, 1.0px], [18, 2.0px]]
+            major_track:
+                filter:
+                    any:
+                        - { kind_detail: track, surface: [paved, asphalt, concrete, gravel] }
+                        - { kind_detail: track, tracktype: [grade1, grade2] }
+                draw:
+                    lines:
+                        color: [1.0,1.0,1.0]
```

Comment:
IMO it's slightly clearer to just say `color: white` for these cases, but totally a matter of preference.